lib/generators/rspec/scaffold/templates/routing_spec.rb

```diff
@@ -5,41 +5,41 @@
   describe "routing" do
 <% unless options[:singleton] -%>
     it "routes to #index" do
-      expect(:get => "/<%= ns_table_name %>").to route_to("<%= ns_table_name %>#index")
+      expect(get: "/<%= ns_table_name %>").to route_to("<%= ns_table_name %>#index")
     end
 
 <% end -%>
 <% unless options[:api] -%>
     it "routes to #new" do
-      expect(:get => "/<%= ns_table_name %>/new").to route_to("<%= ns_table_name %>#new")
+      expect(get: "/<%= ns_table_name %>/new").to route_to("<%= ns_table_name %>#new")
     end
 
 <% end -%>
     it "routes to #show" do
-      expect(:get => "/<%= ns_table_name %>/1").to route_to("<%= ns_table_name %>#show", :id => "1")
+      expect(get: "/<%= ns_table_name %>/1").to route_to("<%= ns_table_name %>#show", :id => "1")
     end
 
 <% unless options[:api] -%>
     it "routes to #edit" do
-      expect(:get => "/<%= ns_table_name %>/1/edit").to route_to("<%= ns_table_name %>#edit", :id => "1")
+      expect(get: "/<%= ns_table_name %>/1/edit").to route_to("<%= ns_table_name %>#edit", :id => "1")
     end
 
 <% end -%>
 
     it "routes to #create" do
-      expect(:post => "/<%= ns_table_name %>").to route_to("<%= ns_table_name %>#create")
+      expect(post: "/<%= ns_table_name %>").to route_to("<%= ns_table_name %>#create")
     end
 
     it "routes to #update via PUT" do
-      expect(:put => "/<%= ns_table_name %>/1").to route_to("<%= ns_table_name %>#update", :id => "1")
+      expect(put: "/<%= ns_table_name %>/1").to route_to("<%= ns_table_name %>#update", :id => "1")
     end
 
     it "routes to #update via PATCH" do
-      expect(:patch => "/<%= ns_table_name %>/1").to route_to("<%= ns_table_name %>#update", :id => "1")
+      expect(patch: "/<%= ns_table_name %>/1").to route_to("<%= ns_table_name %>#update", :id => "1")
```

Comment:
This is mixing the old and the new hash style syntaxes.

Reply:
@pirj I have updated the files according to your feedback and you are correct, robocop of my project complains about the old hash tag and I had to change every time.